.github/workflows/pr.yml

```diff
@@ -90,3 +90,65 @@ jobs:
       with:
         report_paths: unittests.xml
 
+
+  release-build:
+
+    name: Make Release Build
+
+    runs-on: macos-13
+    timeout-minutes: 30
+
+    steps:
+
+    - name: Register SSH keys for access to certificates
+      uses: webfactory/ssh-agent@v0.7.0
+      with:
+        ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY_FASTLANE_MATCH }}
+
+    - name: Check out the code
+      uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Set cache key hash
+      run: |
+        has_only_tags=$(jq '[ .object.pins[].state | has("version") ] | all' DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved)
+        if [[ "$has_only_tags" == "true" ]]; then
+          echo "cache_key_hash=${{ hashFiles('DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved') }}" >> $GITHUB_ENV
+        else
+          echo "Package.resolved contains dependencies specified by branch or commit, skipping cache."
+        fi
+
+    - name: Cache SPM
+      if: env.cache_key_hash
+      uses: actions/cache@v3
+      with:
+        path: DerivedData/SourcePackages
+        key: ${{ runner.os }}-spm-release-${{ env.cache_key_hash }}
+        restore-keys: |
+          ${{ runner.os }}-spm-release-
+
+    - name: Install xcbeautify
+      run: brew install xcbeautify
+
+    - name: Select Xcode
+      run: sudo xcode-select -s /Applications/Xcode_$(<.xcode-version).app/Contents/Developer
+
+    - name: Prepare fastlane
+      run: bundle install
+
+    - name: Archive and upload the app
```

Comment:
Assuming this was a copy-paste issue, but we don’t need `upload` in the name here.

Reply:
Sure, fixed to say "Build the app", as it isn't even creating an archive.